prometheus/exporter.go

```diff
@@ -147,6 +150,30 @@ func (e *Exporter) InitializeMetrics() {
 		[]string{"group_id", "member_count", "protocol", "protocol_type", "state", "coordinator_id"},
 		nil,
 	)
+	// Group Empty Memmbers
+	e.consumerGroupMembersEmpty = prometheus.NewDesc(
+		prometheus.BuildFQName(e.cfg.Namespace, "kafka", "consumer_group_empty_members"),
+		"Consumer Group Empty Members. "+
+			"It will report the number of members in the consumer group with no partition assigned",
```

Comment:
```suggestion
		"It will report the number of members in the consumer group with no partition assigned",
```

I think this does not provide much value in the help text as the given name already says so

Reply:
Agree, fixed